File: 68/68.java
    private static String solution(String line) {
        String[] s = line.split(",");
        int n = s.length;
        int[] a = new int[n];
        for (int i = 0; i < n; i ++) {
            a[i] = Integer.parseInt(s[i]);
        }
        int[] bit = new int[32];
        for (int i = 0; i < n; i ++) {
            for (int j = 0; j < 32; j ++) {
                bit[j] += (1 << j) & a[i];
            }
        }
        int ans = 0;
        for (int i = 0; i < 32; i ++) {
            if (bit[i] % 3 != 0) {
                ans |= (1 << i);
            }
        }
        return ans + "";
    }

File: 68/68.py
def solution(line):
    a=[int(x) for x in line.split(',')]
    n=len(a)
    a.sort()
    i=0
    while i<n:
        if i<n-1 and a[i]!=a[i+1]:
            return a[i]
        i+=3
    return a[n-1]
